### DISCUSSION TITLE
chakraStyles does not exist on type for a select component

### DISCUSSION THREAD

**Discussion by PatFawbertMills** [csandman/chakra-react-select#285] [Q&A] [answered]
I assume I must be doing something wrong with the import of the component? Using VSCode the type checker is flagging:

```
Property 'chakraStyles' does not exist on type 'IntrinsicAttributes & Omit<PublicBaseSelectProps<any, true, GroupBase<any>>, StateManagedPropKeys> & Partial<...> & StateManagerAdditionalProps<...> & RefAttributes<...>'
```

Which is strange as the styles are applying to the component when viewed in the browser.

```
import { Select as CRSelect, ChakraStylesConfig } from 'chakra-react-select'
```
```
const chakraStyles: ChakraStylesConfig = {
    container: (provided, _state) => ({
      ...provided,
      background: 'white',
    }),
  }
```
```
<CRSelect
    chakraStyles={chakraStyles}
    isClearable={false}
    isMulti={true}
    isSearchable={true}
    options={options}
    {...field}
/>
```

**Comment by PatFawbertMills** [comment]
I am also seeing the same typing error for properties `size` too, but again it is applying it correctly when viewed in the browser.

```
Property 'size' does not exist on type 'IntrinsicAttributes & Omit<PublicBaseSelectProps<any, true, GroupBase<any>>, StateManagedPropKeys> & Partial<...> & StateManagerAdditionalProps<...> & RefAttributes<...>'.```

**Comment by csandman** [comment, marked as answer]
This is a known issue at the moment that we're currently working towards fixing. It has to do with a missing type export for projects using `moduleResolution: bundler` (or other newer module resolution types). See #273 for full details.

**Reply by PatFawbertMills** [reply]
Thanks! I appreciate the reply and your work on chakra-react-select

**Reply by csandman** [reply]
No problem! I actually released a new version yesterday that should fix the issue!
